javascripts/content_scripts/posting.app.js

```diff
@@ -1,16 +1,12 @@
 /**
- * @fileOverview This file handles the communication between
+ * fileOverview - This file handles the communication between
```

Comment:
It looks like this might have been a capitalization issue, but the directive is correct. http://usejsdoc.org/tags-file.html